src/ImageRunModal.jsx

```diff
@@ -844,8 +844,19 @@ export class ImageRunModal extends React.Component {
                             </Flex>
                         </FormGroup>
 
-                        { this.state.image && this.state.image.isSystem &&
-                            <FormGroup fieldId='run-image-cpu-priority' label={_("CPU shares")}>
+                        {this.state.owner === systemOwner &&
+                            <FormGroup
+                              fieldId='run-image-cpu-priority'
+                              label={_("CPU shares")}
+                              labelIcon={
+                                  <Popover aria-label={_("CPU Shares help")}
+                                      enableFlip
+                                      bodyContent={_("CPU shares determine priority of running containers. Default priority is 1024. A higher number prioritize this container. Lower numbers decrease priority.")}>
```

Comment:
The statements should be internally consistent with each other.

For example:

```suggestion
                                      bodyContent={_("CPU shares determine the priority of running containers. Default priority is 1024. A higher number prioritizes this container. A lower number decreases priority.")}>
```

Alternate:

```suggestion
                                      bodyContent={_("CPU shares determine the priority of running containers. Default priority is 1024. Higher numbers prioritize this container. Lower numbers decrease the priority of this container.")}>
```

(FWIW: I like the first option better.)